git-dependencies

```diff
@@ -56,12 +62,20 @@ def call_command(args):
 	elif (args.command == 'set-os-filter'):
 		args.arg1 = [x.strip().lower() for x in args.arg1.split(',')] if args.arg1 != '' else []
 		repo.setOSFilter(path = args.arg0, osFilter = args.arg1)
+	elif (args.command == 'dump-deps'):
+		if (args.arg0 == ''):
+			args.arg0 = '*'
+		depStore = {}
+		repo.dumpDeps(path = args.arg0, recursive = args.recursive, depStore = depStore)
+		print(json.dumps(depStore))
 
 def create_parser():
 	parser = argparse.ArgumentParser(prog='git dependencies',
 									 description='Manage dependencies in git repositories',
 									 formatter_class=argparse.RawTextHelpFormatter)
 	parser.add_argument('-r', '--recursive', action='store_true', help='recurse into dependencies')
+	parser.add_argument('-o', '--overrides', help='override dependencies from file')
```

Comment:
Inconsistency between README and the implementation. In the readme the long version of the option is "--override", here it is "--overrides". Please fix.